KILTprotocol/sdk-js
# feat: export did utils

## feat https://github.com/KILTprotocol/ticket/issues/67
Export did utils for use in the universal resolver.

## How to test:
- Create a local project
- Link this sdk branch to it
- Check that the exported functions are available

## Checklist:
